Policies/Kubernetes/enforce-apparmor-profile/rego/src_test.rego

```diff
@@ -0,0 +1,161 @@
+    results := violation with input as input
+    count(results) == 2
+}
+
+test_input_apparmor_containers_allowed_in_list_mixed_no_annotation {
+    input := { "review": input_review_containers_missing_annotation, "parameters": input_parameters_in_list}
+    results := violation with input as input
+    count(results) == 1
+}
+
+test_input_apparmor_containers_not_allowed_not_in_list_mixed_no_annotation {
+    input := { "review": input_review_containers_missing_annotation, "parameters": input_parameters_not_in_list}
+    results := violation with input as input
+    count(results) == 2
+}
+
+test_input_apparmor_containers_not_allowed_not_in_list_mixed {
+    input := { "review": input_review_containers_mixed, "parameters": input_parameters_in_list}
+    results := violation with input as input
+    count(results) == 1
+}
+
+input_review_container = {
+    "object": {
+        "metadata": {
+            "name": "nginx",
+            "annotations": {
+                "container.apparmor.security.beta.kubernetes.io/nginx": "runtime/default"
+            }
+        },
+        "spec": {
+            "containers": [{
+                "name": "nginx",
+                "image": "nginx"
+            }]
+        }
+    }
+}
+
+
+input_review_no_annotation = {
+    "object": {
+        "metadata": {
+            "name": "nginx"
+        },
+        "spec": {
+            "containers": [{
+                "name": "nginx",
+                "image": "nginx"
+            }]
+        }
+    }
+}
+
+input_review_containers = {
+    "object": {
+        "metadata": {
+            "name": "nginx",
+            "annotations": {
+                "container.apparmor.security.beta.kubernetes.io/nginx": "runtime/default",
+                "container.apparmor.security.beta.kubernetes.io/nginx2": "runtime/default"
+            }
+        },
+        "spec": {
+            "containers": two_containers
+        }
+    }
+}
+
+input_review_containers_missing_annotation = {
+    "object": {
+        "metadata": {
+            "name": "nginx",
+            "annotations": {
+                "container.apparmor.security.beta.kubernetes.io/nginx": "runtime/default"
+            }
+        },
+        "spec": {
+            "containers": two_containers
+        }
+    }
+}
+
+input_review_containers_mixed = {
+    "object": {
+        "metadata": {
+            "name": "nginx",
+            "annotations": {
+                "container.apparmor.security.beta.kubernetes.io/nginx": "runtime/default",
+                "container.apparmor.security.beta.kubernetes.io/nginx2": "unconfined"
+            }
+        },
+        "spec": {
+            "containers": two_containers
+        }
+    }
+}
+
+two_containers = [{
+    "name": "nginx",
+    "image": "nginx"
+},{
+    "name": "nginx2",
+    "image": "nginx"
+}]
+
+input_parameters_empty = {
+    "allowedProfiles": []
+}
+
+input_parameters_in_list = {
```

Comment:
Can you please add one more test for multiple items in the input parameters list and initContainers? 
For example:
1. Input parameters with two list items. Say, runtime/default, localhost/sample-profile.
2. Pod with two containers - one init container, one normal container. Init container using localhost profile and normal container using runtime profile.